confidential-data-hub/overlay-network/src/overlay_network/nebula.rs

```diff
@@ -0,0 +1,230 @@
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+use crate::{OverlayNetworkError, Result};
+use crate::overlay_network::config_templates::{WORKER_CONFIG_TEMPLATE, LIGHTHOUSE_CONFIG_TEMPLATE};
+use kms::{Annotations, ProviderSettings};
+use nix::ifaddrs::getifaddrs;
+use nix::sys::socket::{AddressFamily, SockaddrLike};
+use serde::{Deserialize, Serialize};
+use serde_json::Value;
+use serde_yml;
+use std::fs;
+use std::net::Ipv4Addr;
+use std::process::Command;
+
+pub struct NebulaMesh {
+    pod_name: String,
+    lighthouse_ip: String,
+}
+
+// FIXME This should be a shared struct, if possible, with trustee's nebula
+//       plugin. It's that plugin's custom protocol.
```

Comment:
Not really possible. Even our attesters have structs that need to match the verifiers in the AS